src/vs/base/common/winjs.base.d.ts

```diff
@@ -106,7 +106,7 @@ export declare class TPromise<V> {
 	public static wrap<ValueType>(value: Thenable<ValueType>): TPromise<ValueType>;
 	public static wrap<ValueType>(value: ValueType): TPromise<ValueType>;
 
-	public static wrapError<ValueType>(error: any): TPromise<ValueType>;
+	public static wrapError<ValueType>(error: Error): TPromise<ValueType>;
```

Comment:
I met some compile errors I couldn't resolve, I think due to `Thenable` / `TPromise` / `PPromise` all being different types. If it's okay I would like to keep this PR at only the dont-reject-with-strings change.